donut/modules/newsgroups/routes.py

```diff
@@ -178,4 +179,16 @@ def posting_positions(group_id):
     if 'username' not in flask.session:
         return flask.abort(403)
     user_id = auth_utils.get_user_id(flask.session['username'])
-    return flask.jsonify(helpers.get_posting_positions(group_id, user_id))
+    if auth_utils.is_admin():
+        pos = [{
+            'pos_id': groups.get_position_id('Devteam', 'Devteam Member'),
+            'pos_name': 'Devteam Member'
+        }]
+    pos.extend(helpers.get_posting_positions(group_id, user_id))
```

Comment:
It looks like `pos` will be undefined here if the user is not an admin. Make sure to test everything as a non-admin as well.